google-recaptcha.html

```diff
@@ -26,6 +28,9 @@
     :host {
       display: block;
     }
+    :host.invalid {
+      outline: 1px solid red;
```

Comment:
This just puts a red box around the google-recaptcha element when it has the "invalid" class. Ideally we would be able to toggle the rc-anchor-error class on the .rc-anchor element inside the iframe (to get the checkbox inside the captcha to be rendered in red), but not sure if we can access that element due to cross-origin policy.